### ISSUE TITLE
Copy from Git Bash is wrapped since 2.41.0

### ISSUE THREAD

**Issue by placaze** [git-for-windows/git#4463]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.41.0.windows.1
cpu: x86_64
built from commit: ff94e79c4724635915dbb3d4ba38f6bb91528260
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [version 10.0.19045.2965]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VIM
Custom Editor Path:
Default Branch Option:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

No.

### Details

I use Git Bash to consult the logs of my Kubernetes pods with `kubectl logs`, and I copy/paste them sometimes to Notepad++ or Visual Studio Code.
When I copy a part of the `kubectl` output with Git 2.40.1 (and before), lines are not wrapped when pasted. ✔️ 
When I copy a part of the `kubectl` output with Git 2.41.0, lines are wrapped when pasted. ❌ 

I don't know if it's _mintty_ related or not.

![image](https://github.com/git-for-windows/git/assets/57751243/19976939-2870-4951-8e9b-aa474cdabed8)

**Comment by dscho** [comment]
Does this also happen with a regular [MSYS2](https://msys2.org/)?

**Comment by placaze** [comment]
Yes, it also happens with "[msys2-x86_64-20230526.exe](https://github.com/msys2/msys2-installer/releases/download/2023-05-26/msys2-x86_64-20230526.exe)".

The weird part is the behaviour is OK if I `cat` the content of a file and then copy/paste it. I don't know what makes `kubectl` special in this case.

**Comment by dscho** [comment]
Well, `cat.exe` is an MSYS program that directly accesses the emulated Unix pseudo terminal. I bet `kubectl.exe` isn't, and doesn't.

Would you mind raising this issue at https://github.com/msys2/MSYS2-packages/issues/new ?

**Comment by placaze** [comment]
I just did it: https://github.com/msys2/MSYS2-packages/issues/3836

You want me to close the issue here? (feel free to do it if you want)
